packages/content-management/favorites/favorites_server/src/favorites_saved_object.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema } from '@kbn/config-schema';
+import type { SavedObjectsType } from '@kbn/core/server';
+
+export interface FavoritesSavedObjectAttributes {
+  userId: string;
+  type: string;
+  favoriteIds: string[];
+}
+
+const schemaV1 = schema.object({
+  userId: schema.string(),
+  type: schema.string(), // object type, e.g. dashboard
+  favoriteIds: schema.arrayOf(schema.string()),
+});
+
+export const favoritesSavedObjectType: SavedObjectsType = {
+  name: 'favorites',
+  hidden: true,
+  namespaceType: 'single',
+  mappings: {
+    dynamic: false,
+    properties: {
+      userId: {
+        type: 'keyword',
+      },
+    },
+  },
```

Comment:
Do we need to add the `type` to the mappings? Or we don't need it yet?
Update: Oh! I see we only refer to it by ID 👌

Reply:
We actually don't even need userId in the mapping yet because we refer to it by id. I think it is worth to also removing userId from the mapping then, right?

Reply:
IMO, if we foresee them being added to the mappings _soonish_, I'd add them now... Otherwise, we'll have to reprocess/backfill with a noop transform all the existing documents for ES to index them and make them queryable.

cc @jloleysens to keep me honest.

Reply:
>  if we foresee them being added to the mappings soonish,

No particular plans yet